.github/workflows/release-ci.yml

```
@@ -33,99 +33,63 @@ jobs:
 
     - name: Upload release build (darwin-arm64)
       id: upload-release-asset-darwin-arm64
-      uses: actions/upload-release-asset@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      uses: softprops/action-gh-release@v1
       with:
-        upload_url: ${{ github.event.release.upload_url }}
-        asset_path: ./csharp-${{ env.VERSION }}-darwin-arm64.vsix
-        asset_name: csharp-${{ env.VERSION }}-darwin-arm64.vsix
-        asset_content_type: application/zip
+        token: ${{ secrets.GITHUB_TOKEN }}
+        files: ./csharp-${{ env.VERSION }}-darwin-arm64.vsix
```

Maybe we can do it all in one go.

```suggestion
        files: |
          ./csharp-${{ env.VERSION }}-darwin-arm64.vsix
          ./csharp-${{ env.VERSION }}-darwin-x64.vsix
          ./csharp-${{ env.VERSION }}-linux-arm64.vsix
          ./csharp-${{ env.VERSION }}-linux-x64.vsix
          ./csharp-${{ env.VERSION }}-alpine-arm64.vsix
          ./csharp-${{ env.VERSION }}-alpine-x64.vsix
          ./csharp-${{ env.VERSION }}-win32-arm64.vsix
          ./csharp-${{ env.VERSION }}-win32-ia32.vsix
          ./csharp-${{ env.VERSION }}-win32-x64.vsix
```